src/screens/SendImage.tsx

```diff
@@ -68,6 +73,10 @@ const SendImage: React.FC = () => {
     );
   }, []);
 
+  React.useEffect((): void => {
+    setError(false);
```

Comment:
Add a condition to check if the length is ```0``` to insure that this will only run when there is a valid link

Reply:
Fixed in [2a8ff0e](https://github.com/Chillandchat/mobile/pull/84/commits/2a8ff0e8fcefe6453df22d479f477aeaf8544517)